public/model/layerRenderController.ts

```diff
@@ -202,9 +202,10 @@ const getGlobalStates = (mapState: MapState, dashboardProps?: DashboardProps): M
 export const handleBaseLayerRender = (
   layer: MapLayerSpecification,
   maplibreRef: MaplibreRef,
-  onError: Function
+  onError: Function,
+  services: MapServices
```

Comment:
nit: lets keep order of params consistent. i prefer services before onError

Reply:
Sure, just updated.